integration/tests/cook/test_basic.py

```diff
@@ -319,6 +319,57 @@ def test_max_runtime_exceeded(self):
         finally:
             util.kill_jobs(self.cook_url, [job_uuid])
 
+    def memory_limit_exceeded_helper(self, executor_type):
+        command = 'python3 -c ' \
```

Comment:
Can you use a triple quote string for this?

Reply:
Triple quotes will generate a multiline string whereas we want this string to be a single line.

Reply:
Because I am invoking the `python3` command to run it